src/components/Permissions/AppPermissions/AppPermissions.jsx

```diff
@@ -10,23 +10,24 @@ import AppIcon from 'cozy-ui/transpiled/react/AppIcon'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 import IconButton from 'cozy-ui/transpiled/react/IconButton'
 import PreviousIcon from 'cozy-ui/transpiled/react/Icons/Previous'
-import { OpenappButton } from '../OpenappButton'
-import { AboutButton } from '../AboutButton'
-import withAllLocales from '../../../lib/withAllLocales'
+import { OpenappButton } from 'components/Permissions/OpenappButton'
+import { AboutButton } from 'components/Permissions/AboutButton'
+import withAllLocales from 'lib/withAllLocales'
 import NavigationList from 'cozy-ui/transpiled/react/NavigationList'
 import CozyClient, {
   Q,
   useQuery,
   isQueryLoading,
   hasQueryBeenLoaded
 } from 'cozy-client'
-import AccessRightsSection from '../AccessRightsSection'
-import LatestDataReleases from '../LatestDataReleases'
+import AccessRightsSection from 'components/Permissions/AccessRightsSection'
+import LatestDataReleases from 'components/Permissions/LatestOutgoingDataHistory'
```

Comment:
I just noticed that `LatestDataReleases` name mismatches the `LatestOutgoingDataHistory` component name. Is that intended?